applications/mne_analyze/mne_analyze/mainwindow.cpp

```diff
@@ -77,57 +78,56 @@ using namespace DISPLIB;
 // DEFINE MEMBER METHODS
 //=============================================================================================================
 
-MainWindow::MainWindow(QSharedPointer<ANSHAREDLIB::PluginManager> pPluginManager,
-                       QWidget *parent)
+MainWindow::MainWindow(AnalyzeCore* pAnalyzeCore, QWidget *parent)
```

Comment:
Our [coding conventions](https://mne-cpp.github.io/pages/development/contr_style.html) ask for aligning function parameters underneath each other.